agents/craftassist/craftassist_agent.py

```diff
@@ -87,6 +88,13 @@ def __init__(self, opts):
         ]
         self.perceive_on_chat = True
 
+        # Add optional logging for timeline
+        if opts.log_timeline:
```

Comment:
Could you move this part to `loco_mc_agent.py`? similar to how to initialize db log file in `sql_memory.py` instead of `mc_memory.py` https://github.com/facebookresearch/droidlet/blob/amanda/timeline/droidlet/memory/sql_memory.py#L89